thumbor/engines/extensions/exif_orientation_editor.py

```diff
@@ -17,7 +17,10 @@ class ExifOrientationEditor:
 
     def __init__(self, data):
         if data[:6] != EXIF_HEADER:
-            raise SyntaxError("not a TIFF file (header %r not valid)" % data[:4])
+            raise SyntaxError(
+                "not a TIFF file (header %r not valid)"  # pylint: disable=consider-using-f-string
```

Comment:
Is there any reason not to use _f string_ here?

Reply:
Not really. Fixed! 👍